src/cloud-cli/applications/list-apps.ts

```diff
@@ -27,10 +22,12 @@ export async function listApps(host: string, port: string) {
       logger.info("no application found");
       return;
     }
+    const formattedData: Application[] = []
     for (const application of data) {
       // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
-      console.log({ Name: application.Name, ID: application.ID, Status: application.Status });
+      formattedData.push({ "Name": application.Name, "ID": application.ID, "Status": application.Status, "MaxVMs": application.MaxVMs });
     }
+    console.log(JSON.stringify(formattedData));
```

Comment:
Eventually we will come up with a nice way to present the results of these commands. For now its useful to have parsable JSON.